File: CloudBridge-WD/src/main/java/de/r3pt1s/CloudBridge/network/protocol/packet/UnregisterServerPacket.java
// This class was created by r3pt1s
package de.r3pt1s.CloudBridge.network.protocol.packet;

public class UnregisterServerPacket extends Packet {

    public Object name = "";

    @Override
    public void encode() {
        super.encode();
        put(name);
    }

    @Override
    public void decode() {
        super.decode();
        name = get();
    }

    @Override
    public Integer getId() {
        return ID_UNREGISTER_SERVER;
    }

    public static UnregisterServerPacket create(String name) {
        UnregisterServerPacket pk = new UnregisterServerPacket();
        pk.name = name;
        return pk;
    }
}

File: CloudSystem/src/Cloud/network/protocol/packet/UnregisterServerPacket.php
<?php

namespace Cloud\network\protocol\packet;

class UnregisterServerPacket extends Packet {

    public string $name = "";

    public function encode(): void {
        parent::encode();
        $this->put($this->name);
    }

    public function decode(): void {
        parent::decode();
        $this->name = $this->get();
    }

    public function getId(): int {
        return self::ID_UNREGISTER_SERVER;
    }

    public static function create(string $name): self {
        $pk = new self;
        $pk->name = $name;
        return $pk;
    }
}